### PR TITLE
[WIP] adding basic unitest

### PR DESCRIPTION
**Pull request by skoudoro**
Adding basic tests using pytest

### CODE REVIEW

**Comment by ghoshbishakh** [comment]
Wow, huge changes 😄

File: website/tests/test_website_forms.py [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+
+    # Test Valid Data
+    form = EditFixedSectionForm({
+        'title': 'My test',
+        'body_markdown': 'This is a test!',
+    })
+    assert form.is_valid()
+
+    section = form.save()
+    assert section.title == 'My test'
```
Comment (ghoshbishakh) [inline website/tests/test_website_forms.py]: aren't these django functionalities? and django is already well tested.

Comment (skoudoro) [inline website/tests/test_website_forms.py]: I get some doubt too when I created form's tests. Do you know any other way to test forms or should we test them only through view ?

File: website/tests/test_website_models.py [hunk 2]
```diff
@@ -0,0 +1,191 @@
...
+import time
+from distutils.version import LooseVersion
+from django.utils import timezone
+from ..models import Profile, WebsiteSection, NewsPost, \
+    Publication, CarouselImage, DocumentationLink
+
+
+def test_string_representation(random_user):
+    entry = WebsiteSection(title="My entry title")
+    assert str(entry) == entry.title
```
Comment (ghoshbishakh) [inline website/tests/test_website_models.py]: this is also a django function right? So I think django already has the relevant tests. I am also new to testing in django so I am not sure.

Comment (skoudoro) [inline website/tests/test_website_models.py]: You override __str__() function on each model so I test we get the desired value that you define. So I think it's fine here.

File: website/tests/test_website_forms.py (hunk 1 above)
Comment (ghoshbishakh) [inline website/tests/test_website_forms.py]: @skoudoro I am also not sure. Have you seen any similar django repo that has tests?

File: website/tests/test_website_models.py (hunk 2 above)
Comment (ghoshbishakh) [inline website/tests/test_website_models.py]: @skoudoro Ok

**Pull request merged**